terracumber/terraformer.py

```diff
@@ -66,12 +66,12 @@ def inject_repos(self, custom_repositories_json):
                     node_mu_repos = repos.get(node, None)
                     replacement_list = ["additional_repos = {"]
                     for name, url in node_mu_repos.items():
-                        replacement_list.append(f'\n{" " * 4}{name} = "{url}"')
+                        replacement_list.append(f'\n    "{name}" = "{url}"')
                     replacement_list.append("\n  }")
```

Comment:
I think, we can also skip the witespaces here, to get the correct indentation for the closing bracket.
```suggestion
                    replacement_list.append("\n}")
```

```hcl
additional_repos = {
    "35083" = "http://download.suse.de/ibs/SUSE:/Maintenance:/35083/SUSE_Updates_SLE-Module-SUSE-Manager-Server_4.3_x86_64/"
    "34966" = "http://download.suse.de/ibs/SUSE:/Maintenance:/34966/SUSE_Updates_SLE-Product-SUSE-Manager-Server_4.3_x86_64/"
  }
```
vs.
```hcl
additional_repos = {
    "35083" = "http://download.suse.de/ibs/SUSE:/Maintenance:/35083/SUSE_Updates_SLE-Module-SUSE-Manager-Server_4.3_x86_64/"
    "34966" = "http://download.suse.de/ibs/SUSE:/Maintenance:/34966/SUSE_Updates_SLE-Product-SUSE-Manager-Server_4.3_x86_64/"
}
```